Data/models/DataController.swift

```diff
@@ -26,15 +26,11 @@ public class DataController: NSObject {
     public static var shared: DataController = DataController()
     public static var sharedInMemory: DataController = InMemoryDataController()
     
-    public func storeExists() -> Bool {
-        return FileManager.default.fileExists(atPath: storeURL.path)
-    }
-    
-    public func migrateToNewPathIfNeeded() {
+    public func migrateToNewPathIfNeeded() throws {
         func sqliteFiles(from url: URL, dbName: String) throws -> [URL] {
             return try FileManager.default.contentsOfDirectory(at: url, includingPropertiesForKeys: nil, options: []).filter({$0.lastPathComponent.hasPrefix(dbName)})
         }
-        if FileManager.default.fileExists(atPath: oldStoreURL.path) && !storeExists() {
+        if FileManager.default.fileExists(atPath: oldStoreURL.path) {
```

Comment:
Not sure if we want to force over-ride existing data. This would replace all of our current user's new bookmarks, history items, tabs, etc...

Talking with product on how they want to handle this.